index.js

```diff
@@ -1 +1,40 @@
-console.log('Hello world!');
+const os = require('os');
+const fs = require('fs');
+const childProcess = require('child_process');
+
+const WINDOWS_COMMAND = `powershell "Get-Process | Sort-Object CPU -Descending | Select-Object -Property Name, CPU, WorkingSet -First 1 | ForEach-Object { $_.Name + ' ' + $_.CPU + ' ' + $_.WorkingSet }"`;
+const UNIX_COMMAND = 'ps -A -o %cpu,%mem,comm | sort -nr | head -n 1';
+
+const isWindows = os.type() === 'Windows_NT';
+const command = isWindows ? WINDOWS_COMMAND : UNIX_COMMAND;
+
+// Function to get the most CPU-intensive running application
+function getMostCpuIntensiveApplication(callback) {
+  childProcess.exec(command, (error, stdout) => {
+    if (error) {
+      callback(`${Math.floor(Date.now() / 1000)} : Error: ${error}`);
+    } else {
+      callback(`${Math.floor(Date.now() / 1000)} : ${stdout}`);
```

Comment:
if you trim the stdout `stdout.trim()` it wiil fix the problem to output to new line.